### ISSUE TITLE
Accumulating rpc_clnt_submit error in glustershd.log

### ISSUE THREAD

**Issue by wdbasson** [openshift/openshift-ansible#10273]
#### Description

On an OKD v3.9 and v3.10 cluster with CNS Gluster, deleting PVCs from the file based glusterfs storageclass causes accumulation of rpc_clnt_submit errors in /var/log/glusterfs/glustershd.log. Over time all available CPU on the storage nodes are consumed by these errors (presumably attempting to establish accumulating rpc connections).

##### Version

```
[wdbasson@navi openshift-ansible]$ ansible --version
ansible 2.4.3.0
  config file = /home/wdbasson/terraform-libvirt/openshift-ansible/ansible.cfg
  configured module search path = [u'/home/wdbasson/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15 (default, Jun 27 2018, 13:05:28) [GCC 8.1.1 20180531]
[wdbasson@navi openshift-ansible]$ git describe
openshift-ansible-3.10.51-1-16-g1a911c5d0
```

##### Steps To Reproduce
1. deploy OKD v3.9 or v3.10 using openshift-ansible
2. create a PVC from file-based storage class glusterfs-storage
3. delete PVC
4. tail -f /var/log/glusterfs/glustershd.log
5. repeat steps 2 and 3

##### Expected Results
Deleting PVCs shouldn't accumulate errors and deplete storage node resources. Using openshift_storage_glusterfs_version=gluster3u13_centos7 doesn't cause these errors.

##### Observed Results

Error log from a storage node:
```
[centos@node-0 ~]$ sudo tail -n 5 /var/log/glusterfs/glustershd.log 
[2018-09-29 10:57:40.210212] W [rpc-clnt.c:1753:rpc_clnt_submit] 0-vol_898c14de3760573e31912959d9281898-client-0: error returned while attempting to connect to host:(null), port:0
[2018-09-29 10:57:40.213736] W [rpc-clnt.c:1753:rpc_clnt_submit] 0-vol_898c14de3760573e31912959d9281898-client-1: error returned while attempting to connect to host:(null), port:0
[2018-09-29 10:57:40.214018] W [rpc-clnt.c:1753:rpc_clnt_submit] 0-vol_898c14de3760573e31912959d9281898-client-1: error returned while attempting to connect to host:(null), port:0
[2018-09-29 10:57:40.216782] W [rpc-clnt.c:1753:rpc_clnt_submit] 0-vol_898c14de3760573e31912959d9281898-client-2: error returned while attempting to connect to host:(null), port:0
[2018-09-29 10:57:40.217041] W [rpc-clnt.c:1753:rpc_clnt_submit] 0-vol_898c14de3760573e31912959d9281898-client-2: error returned while attempting to connect to host:(null), port:0
```

##### Additional Information

Reproduced on local libvirt and bare-metal deployment using openshift-ansible releases v3.9 and v3.10 with different images for Gluster and heketi:

```
OS: centos-release-7-5.1804.4.el7.centos.x86_64
openshift_storage_glusterfs_version=gluster4u0_centos7 or latest
openshift_storage_glusterfs_heketi_version=latest, dev or 7
```

Example inventory file for local libvirt deployment:

```
# Create an OSEv3 group that contains the masters and nodes groups
[OSEv3:children]
masters
nodes
etcd
glusterfs
[OSEv3:vars]
openshift_node_groups=[{'name': 'node-config-all', 'labels': ['node-role.kubernetes.io/master=true', 'node-role.kubernetes.io/infra=true', 'node-role.kubernetes.io/compute=true']}]
# SSH user this user should allow ssh based auth without requiring a password
ansible_become=yes
ansible_become_user=root
ansible_ssh_user=centos
openshift_set_node_ip=True
deployment_type=origin
os_firewall_use_firewalld=True
openshift_clock_enabled=True
openshift_disable_check=*
openshift_check_min_host_disk_gb=1
openshift_check_min_host_memory_gb=1
openshift_storage_glusterfs_version=gluster4u0_centos7
openshift_storage_glusterfs_block_deploy=true
openshift_storage_glusterfs_block_host_vol_size=100
openshift_storage_glusterfs_block_storageclass=true
openshift_storage_glusterfs_block_storageclass_default=false
openshift_hosted_prometheus_deploy=false
openshift_metrics_install_metrics=false
openshift_logging_install_logging=false
openshift_master_htpasswd_users={'foo': 'bar'}
[masters]
node-0.openshift.local
node-1.openshift.local
node-2.openshift.local
[etcd]
node-0.openshift.local
node-1.openshift.local
node-2.openshift.local
[nodes]
node-0.openshift.local openshift_ip=192.168.100.239 openshift_public_ip=192.168.100.239 openshift_node_group_name="node-config-all"
node-1.openshift.local openshift_ip=192.168.100.50 openshift_public_ip=192.168.100.50 openshift_node_group_name="node-config-all"
node-2.openshift.local openshift_ip=192.168.100.17 openshift_public_ip=192.168.100.17 openshift_node_group_name="node-config-all"
[glusterfs]
node-0.openshift.local glusterfs_ip=192.168.100.239 glusterfs_devices='[ "/dev/vdb" ]'
node-1.openshift.local glusterfs_ip=192.168.100.50 glusterfs_devices='[ "/dev/vdb" ]'
node-2.openshift.local glusterfs_ip=192.168.100.17 glusterfs_devices='[ "/dev/vdb" ]'
```

**Comment by wdbasson** [comment]
This is now also happening on the latest gluster/gluster-centos:gluster3u12_centos7 image.

**Comment by wdbasson** [comment]
I can confirm this issue is resolved on OKD v3.11 with the latest gluster image: gluster/gluster-centos:latest
